lib/mongoid/contextual/atomic.rb

```diff
@@ -52,6 +52,18 @@ def inc(incs)
         view.update_many("$inc" => collect_operations(incs))
       end
 
+      # Perform an atomic $mul operation on the matching documents.
+      #
+      # @example Perform the atomic multiplication.
+      #   context.set_mul(likes: 10)
+      #
+      # @param [ Hash ] factors The operations.
+      #
+      # @return [ nil ] Nil.
+      def set_mul(factors)
```

Comment:
`mul` I think is fine to be as-is (not `set_mul`). It's analogous to `inc` and is a unique word (unlikely to be used as a field by a model)